.github/workflows/lint.yaml

```diff
@@ -0,0 +1,14 @@
+# Check for lint error and auto correct them
+
+name: Check for lint errors
+
+on: [push]
```

Comment:
not on pull request too?

Reply:
Yes that is correct, even though the user will push it to github before doing a pull request, it is useful for the reviewer to see the check. Thanks! I didn't think of that.